.travis.yml

```diff
@@ -34,7 +33,7 @@ matrix:
         - php: 7.3
           env: DEPENDENCIES="dunglas/symfony-lock:^3"
         - php: 7.3
-          env: DEPENDENCIES="dunglas/symfony-lock:^4"
+          env: DEPENDENCIES="dunglas/symfony-lock:^4" SYMFONY_DEPRECATIONS_HELPER="max[direct]=0"
```

Comment:
this prevents deprecations triggered by the symfony/twig-bundle 4.4 from breaking the build.

Reply:
According to https://symfony.com/doc/current/components/phpunit_bridge.html#direct-and-indirect-deprecations, would `max[self]=0` be a saner value? Also, moving it to the `env.global` would be better, isn't it?

Reply:
right, self seems right. right now it works, but if we trigger our own deprecations, it can stop working.

not sure if we want this to be global. i guess actually yes - then we don't need a config for it at all but can just unconditionally do it. we don't care if other things trigger deprecations, we only care if our own code triggers deprecations.

Reply:
erm, i mean make it global. somehow mixed it up with another thing where i do a composer config for flex :-D

Reply:
changed